src/pages-and-resources/pages/PageCard.jsx

```diff
@@ -1,73 +1,80 @@
-import { Button } from '@edx/paragon';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faCog } from '@fortawesome/free-solid-svg-icons';
 
-import { history } from '@edx/frontend-platform';
+import { Link } from 'react-router-dom';
+import StatusBadge from '../../generic/status-badge/StatusBadge';
 import messages from '../messages';
 import { PagesAndResourcesContext } from '../PagesAndResourcesProvider';
 
 const CoursePageShape = PropTypes.shape({
   id: PropTypes.string.isRequired,
-  title: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
   description: PropTypes.string.isRequired,
-  isEnabled: PropTypes.bool.isRequired,
-  showSettings: PropTypes.bool.isRequired,
-  showStatus: PropTypes.bool.isRequired,
-  showEnable: PropTypes.bool.isRequired,
+  enabled: PropTypes.bool.isRequired,
+  allowed_operations: PropTypes.shape({
+    enable: PropTypes.bool.isRequired,
+    configure: PropTypes.bool.isRequired,
+  }).isRequired,
 });
 
 export { CoursePageShape };
 
-function PageCard({ intl, page }) {
+function PageCard({
+  intl,
+  page,
+}) {
   const { path: pagesAndResourcesPath } = useContext(PagesAndResourcesContext);
-
-  const pageStatusMsgId = page.isEnabled ? 'pageStatus.enabled' : 'pageStatus.disabled';
   const componentClasses = classNames(
-    'd-flex flex-column align-content-stretch',
-    'bg-light-100 p-3 border shadow',
-    { 'border-gray-500': page.isEnabled, 'border-gray-100': !page.isEnabled },
+    'border shadow card',
+    {
+      'border-gray-500': page.enabled,
+      'border-gray-100': !page.enabled,
+    },
   );
 
   const handleClick = () => {
     history.push(`${pagesAndResourcesPath}/${page.id}`);
   };
 
   return (
-    <div
-      className="d-flex flex-column align-content-stretch p-3 col-sm-12 col-md-6 col-lg-4"
+    <Card
+      className={componentClasses}
+      style={{
+        width: '24rem',
+        height: '14rem',
+      }}
     >
-      <div
-        className={componentClasses}
-        style={{
-          flexBasis: '100%',
-        }}
-      >
-        <div className="d-flex flex-row">
-          <span className="font-weight-bold">{page.title}</span>
-          {page.showSettings && <FontAwesomeIcon icon={faCog} className="ml-auto" />}
-        </div>
+      <Card.Body className="d-flex flex-column">
+        <Card.Title className="d-flex mb-0 align-items-center justify-content-between">
+          <h4 className="mb-0">{page.name}</h4>
+          {(page.allowed_operations.configure || page.allowed_operations.enable)
+          && (
+            <Link to={`${pagesAndResourcesPath}/${page.id}/settings`} className="mb-0">
+              <Settings size="lg" />
+            </Link>
+          )}
+        </Card.Title>
 
-        <div>
-          {page.showStatus && <span>{intl.formatMessage(messages[pageStatusMsgId])}</span>}
+        <div className="my-1">
+          <StatusBadge status={page.enabled} />
```

Comment:
UI is not matching figma - The badge size is not according to figma design. is this PR not targeting Figma UI?

Reply:
I think the colours were off, but the size does seem to be according to the Figma design. I've fixed the colours.

Reply:
I think the padding of the status badge is also needed to be fixed. The dividers used not part of the Figma design and are going out from both sides of the modal. you can see that in the attached video as well. if that is needed to be fixed in this PR. then you might need to look at it again.

Reply:
I've fixed some of those issues, but I think we might be looking at different designs. I was following this one: 
https://www.figma.com/file/F5QstWQ8YAlStIabotpsIy/%5BHIFI%5D-Apps-and-resources-landing-page?node-id=80%3A73

Reply:
@xitij2000 we are following this Figma design, not sure if you have access to this but please check.
https://www.figma.com/file/SeNpRff5HxzdpCgoD5pmlQ/Apps-and-resources-landing-page